### ISSUE TITLE
Editable Adrenaline Label

### ISSUE THREAD

**Issue by babel2uk** [Genii-Locorum/outgunned#10]
Some of the Action Flicks rename "Adrenaline" to "Qi", would it be feasible to make that label editable on an Actor level (one of the Action Flicks lets you mix and match actors from all action flicks, so it couldn't be a global change)? It would also cater for Adventure in the short term which calls it "Luck" instead.

Best
Matt

**Comment by Genii-Locorum** [comment]
@babel2uk 

This is an easy one Matt.  Added the name as a piece of data to the actor.  It does mean for foreign languages people will need to edit it.  

Another other option would be to make it a translatable drop down but I've avoided this as the player can choose whatever they want and allows for homebrew variants.

![og](https://github.com/user-attachments/assets/36b6bdca-0e20-48ec-b477-4ceef8c4210e)

**Comment by babel2uk** [comment]
Perfect 😀On 21 Sept 2024 15:36, Genii-Locorum ***@***.***> wrote:
@babel2uk
This is an easy one Matt.  Added the name as a piece of data to the actor.  It does mean for foreign languages people will need to edit it.    It can only be edited when the character sheet is unlocked.
Another other option would be to make it a translatable drop down but I've avoided this as the player can choose whatever they want and allows for homebrew variants.
og.jpg (view on web)
